src/component/agenda/agenda.js

```diff
@@ -32,13 +34,13 @@ export class Session extends React.Component {
         this.setState({
             slot
         })
-        this.props.updateState(`${this.props.agenda}`, e)
```

Comment:
just curious why you used a template literal here, were you stringifying an object?